listenbrainz/webserver/views/user.py

```diff
@@ -243,6 +261,122 @@ def playlists(user_name: str):
         user=user
     )
 
+@user_bp.route("/<user_name>/recommendations")
+def recommendations(user_name: str):
```

Comment:
name function to make it clear that this is playlist-related

Reply:
Existing code that I dont want to touch right now. Passing on this.